edi_sale_utm_oca/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2024 Camptocamp
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "EDI Sale utm",
+    "summary": """
+    Allows to set a utm source on an exchange type and sets it when the order is created.
+    """,
+    "version": "16.0.1.0.0",
+    "development_status": "Alpha",
+    "license": "LGPL-3",
+    "website": "https://github.com/OCA/edi-framework",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "depends": ["utm", "edi_sale_oca"],
+    "data": [
+        "data/utm_source.xml",
+        "views/edi_exchange_record_views.xml",
+    ],
+    "installable": True,
+    "auto_install": True,
+}
```

Comment:
Please remove `"auto_install": True`

Add `"maintainers": ["simahawk"],`

Change to `Allows to define an utm source on an exchange type and automatically set it on the sale order when it is created.`